JASP-Engine/JASP/R/cfa.R

```diff
@@ -983,10 +981,10 @@ ConfirmatoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
 
 .cfaSyntax <- function(jaspResults, options, dataset, cfaResult) {
   if (is.null(cfaResult) || !options$showSyntax || !is.null(jaspResults[["syntax"]])) return()
-  
+
   mod <- .optionsToCFAMod(options, dataset, cfaResult, FALSE)
 
   jaspResults[["syntax"]] <- createJaspHtml(mod, class = "jasp-code", position = 7, title = "Model syntax")
-  jaspResults[["syntax"]]$copyDependenciesFromJaspObject(jaspResults[["maincontainer"]][["cfatab"]])
-  jaspResults[["syntax"]]$dependOnOptions("showSyntax")
+  jaspResults[["syntax"]]$dependOn(optionsFromObject = jaspResults[["maincontainer"]][["cfatab"]])
+  jaspResults[["syntax"]]$dependOn("showSyntax")
```

Comment:
This could just be a one-liner:
`jaspResults[["syntax"]]$dependOn("showSyntax", optionsFromObject = jaspResults[["maincontainer"]][["cfatab"]])`
but that's more of a FYI.